api/src/public/login/style.css

```
@@ -147,6 +147,38 @@ form {
   text-decoration: none;
 }
 
+#password-container {
+  position: relative;
+  height: 40px;
+}
+
+#password-container #password {
+  width: 245px;
+  padding: 6px 43px 6px 10px;
+}
+
+#password-container #password-toggle {
+  position: absolute;
+  right: 0px;
+  top: 0px;
+  height: 100%;
+  padding: 0 10px;
+  cursor: pointer;
+}
+
+#password-container.hidden-password .show-password-icon,
+#password-container .hide-password-icon {
+  width: 24px;
+  height: 100%;
+  display: inline-block;
+  vertical-align: middle;
+}
+
+#password-container .show-password-icon,
+#password-container.hidden-password .hide-password-icon {
+  display: none;
+}
+
```

I think lines 169 and 170 can be simplified to being a single css rule instead of two. This is purely for readability...

```suggestion
#password-container img {
  width: 24px;
  height: 100%;
  vertical-align: middle;
}

#password-container.hidden-password .show-password-icon,
#password-container .hide-password-icon {
  display: inline-block;
}

#password-container .show-password-icon,
#password-container.hidden-password .hide-password-icon {
  display: none;
}

```

---

Done